src/Controller/Component/CsrfComponent.php

```diff
@@ -56,7 +56,7 @@ class CsrfComponent extends Component
         'cookieName' => 'csrfToken',
         'expiry' => 0,
         'secure' => false,
-        'httpOnly' => false,
+        'httpOnly' => true,
```

Comment:
I'm actually pretty sure this is not needed, but I'm no security expert. What would be the attack vector?